1-mandatory/2-people-i-know.js

```diff
@@ -394,6 +395,13 @@ This time, I only want the full names ("<firstname> <lastname>") of my friends w
 */
 
 let friendsWhoAreColleaguesOfStacie = [];
+for (let friend of friends) {
+  for (let colleague of friend.colleagues) {
+    if (colleague.name === "Stacie Villarreal") {
```

Comment:
I think here you can use `friend.colleagues.find` like below:

`if (friend.colleagues.find(colleague => colleague.name === "Stacie Villarreal")`

Nothing wrong with your solution :) just an alternate option

[Array.prototype.find()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)

Reply:
thanks berkeli... thats amazing.. i will note that